OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -194,7 +194,11 @@ public boolean isDiscoveryEnabled(@NonNull IEdxEnvironment environment) {
                     environment.getConfig().getDiscoveryConfig().getCourseDiscoveryConfig() != null &&
                     environment.getConfig().getDiscoveryConfig().getCourseDiscoveryConfig().isDiscoveryEnabled() &&
                     environment.getConfig().getDiscoveryConfig().getCourseDiscoveryConfig().isWebviewDiscoveryEnabled()
-            ;
+                    ;
+        }
+
+        public boolean isWebviewDiscoveryEnabled() {
```

Comment:
Type in word `Webview`, it should be `isWebViewDiscoveryEnabled`.

Reply:
Let's leave it for now, cuz all the functions with such name in the class has the same convention.